include/math/seadMatrixCalcCommon.hpp

```diff
@@ -1542,16 +1542,20 @@ void Matrix34CalcCommon<T>::makeSRT(Base& o, const Vec3& s, const Vec3& r, const
     const T cosV[3] = {MathCalcCommon<T>::cos(r.x), MathCalcCommon<T>::cos(r.y),
                        MathCalcCommon<T>::cos(r.z)};
 
+    f32 c0_c2 = cosV[0] * cosV[2];
```

Comment:
```suggestion
    T c0_c2 = cosV[0] * cosV[2];
```
same for the two other variables you're introducing

Reply:
Done.